content/en/dynamic_instrumentation/expression-language.md

```diff
@@ -63,13 +66,6 @@ The following examples use a variable named `myCollection` defined as `[1,2,3]`:
 | `len(value_src)` | Gets the collection size. | {{< expression-language-evaluator expression="len(myCollection)" >}} |
 | `[ n ]` | For collections, returns the nth item in the collection. For maps and dictionaries, returns the value that corresponds to the key `n`. If the item does not exist, the expression yields an error. | {{< expression-language-evaluator expression="myCollection[1]" >}} |
 
-[1]: /metrics/types/?tab=count#metric-types
```

Comment:
Looks like this link is still being referenced (line 64).

Reply:
L64 is an argument to an expression, AFAICT these were orphaned references 🤷

Reply:
Ah ok, thank you for clarifying!